x/programs/runtime/runtime.go

```diff
@@ -52,15 +53,17 @@ func (r *runtime) Create(ctx context.Context, programBytes []byte) (uint64, erro
 	// get programId
 	programID := initProgramStorage()
 
-	// call initialize
+	// call initialize if it exists
 	result, err := r.Call(ctx, "init", uint64(programID))
```

Comment:
I think we should remove this. if the user of the runtime wants to call init they should do so. I think we simplify things a lot if we remove create.

flow
- new runtime
- initialize runtime
- use runtime

create would be logic inside of an action right?

Reply:
yea, most of the logic in `Create` would be inside of an action that calls `initProgramStorage`.  I say for now, we leave this in because 
- we don't have actions
- we need logic for storing the program in state and returning the programID
  - this logic must be called once per runtime. 
  - However, Initialize can be called multiple times, say to initialize program bytes without creating a new `programID`

Reply:
also without a function like `Create` i found myself having to duplicate this logic. Would need to add similar code to all of the `examples` &  this helps with the `simulator` pr as well. I'm  using `Create` in the simulator to simulate the action.

Reply:
fine to leave it for now as its already merged.